fzf-url.sh

```diff
@@ -6,7 +6,7 @@
 #===============================================================================
 
 fzf_filter() {
-    fzf-tmux -d 35% -m -0 --no-preview --no-border
+    fzf-tmux -w 50% -m -0 --no-preview --no-border
```

Comment:
could we make this somehow configurable? and maybe incorporate the height as well? you can use the `-p` flag to start fzf-tmux in a popup and specify width and height after it, respectively (`tmux-fzf -p 50%,60%`)

Reply:
I have created an improved PR that incorporates the ability to set custom options passed to `fzf-tmux`. See #16